airbyte-integrations/connectors/destination-oracle/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'application'
+    id 'airbyte-docker'
+    id 'airbyte-integration-test-java'
+}
+
+application {
+    mainClass = 'io.airbyte.integrations.destination.oracle.OracleDestination'
+}
+
+dependencies {
+
+    // required so that log4j uses a standard xml parser instead of an oracle one (that gets pulled in by the oracle driver)
+    implementation group: 'xerces', name: 'xercesImpl', version: '2.12.1'
+
+    implementation project(':airbyte-db')
+    implementation project(':airbyte-integrations:bases:base-java')
+    implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-integrations:connectors:destination-jdbc')
+
+    implementation "com.oracle.database.jdbc:ojdbc8-production:19.7.0.0"
+
+    testImplementation 'org.apache.commons:commons-lang3:3.11'
+    testImplementation 'org.testcontainers:oracle-xe:1.15.2'
+
+    integrationTestJavaImplementation project(':airbyte-integrations:bases:standard-destination-test')
+    integrationTestJavaImplementation project(':airbyte-integrations:connectors:destination-oracle')
+    integrationTestJavaImplementation project(':airbyte-workers')
```

Comment:
do you still need this dep?

Reply:
Yes, `AirbyteDestination` is declared in that package.